TesseractOCR/build_dependencies.sh

```diff
@@ -135,6 +136,8 @@ rm -rf $GLOBAL_OUTDIR lib include
 git submodule init
 git submodule update
 
+curl http://leptonica.org/source/$LEPTON_VER.tar.gz | tar -xpf-
```

Comment:
I hope this isn't too fragile in that we assume the user has `curl` installed. It seems to come with OSX, though, so I guess it's okay.